pageserver/src/http/openapi_spec.yml

```diff
@@ -932,6 +932,59 @@ paths:
               schema:
                 $ref: "#/components/schemas/ServiceUnavailableError"
 
+  /v1/tenant/{tenant_shard_id}/heatmap_upload:
+    parameters:
+      - name: tenant_shard_id
+        in: path
+        required: true
+        schema:
+          type: string
+    post:
+      description: |
+        If the location is in an attached mode, upload the current state to the remote heatmap
+      responses:
+        "200":
+          description: Success
+        "500":
+          description: Generic operation error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "503":
+          description: Temporarily unavailable, please retry.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ServiceUnavailableError"
+
+  /v1/tenant/{tenant_shard_id}/secondary/download:
```

Comment:
That's a vague endpoint name considering its purpose: There is nothing about the heatmap in the URL so one thinks it's about the entire secondary in general. IMO it should be renamed to `download_heatmap` or `heatmap_download`.

Reply:
As well as the heatmap, it's downloading layer files -- so it is the more general "do the downloads for this secondary location" action, rather than something heatmap specific.